src/server/req_manager.c

```diff
@@ -2689,15 +2699,15 @@ make_host_addresses_list(char *phost, u_long **pul)
 		memmove(tpul->pul, *pul, tpul->len);
 
 		if (hostaddr_tree == NULL ) {
-			hostaddr_tree = cr_tree(0);
+			hostaddr_tree = create_tree(0, 0);
```

Comment:
Please replace the first argument with AVL_NO_DUP_KEYS. This is needed in other create_tree instances too (wherever we don't need duplicates in the tree)